app/javascript/angular/code/services/google/_map.js

```diff
@@ -68,14 +68,15 @@ export const map = (
       googleMaps.listenPanOrZoom(this.mapObj, callback);
     },
 
-    goToAddress: function(address) {
+    goToAddress: function(address, fetch_sessions) {
```

Comment:
I'd suggest calling it `callback`. `fetchSession` is not a concern of google maps

Reply:
Also, watch out for the snake_case in JS